{"name": "Order creation without ingredients but with auth error test", "status": "passed", "start": 1732664533386, "stop": 1732664533749, "uuid": "78308a41-fafc-4205-96f7-d3db6ae4ef39", "historyId": "10732fc5e19da7c4ffd9b151d7ba5684", "testCaseId": "10732fc5e19da7c4ffd9b151d7ba5684", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_no_ingredients_with_auth", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "DESKTOP-V6N82FL"}, {"name": "thread", "value": "24496-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}